winbuild/build_prepare.py

```diff
@@ -386,6 +388,27 @@ def cmd_msbuild(
         ],
         "bins": [r"*.dll"],
     },
+    "libavif": {
+        "url": f"https://github.com/AOMediaCodec/libavif/archive/v{V['LIBAVIF']}.zip",
+        "filename": f"libavif-{V['LIBAVIF']}.zip",
+        "dir": f"libavif-{V['LIBAVIF']}",
```

Comment:
```suggestion
```
This is no longer needed, thanks to https://github.com/python-pillow/Pillow/pull/8567

Reply:
I've created https://github.com/fdintino/Pillow/pull/13